src/ceph-volume/ceph_volume/tests/api/test_lvm.py

```diff
@@ -1,6 +1,5 @@
 import os
 import pytest
-from mock.mock import patch
```

Comment:
Please leave this in, this came via a backport and will be used by a later backport (#33578)

Reply:
done